### DISCUSSION TITLE
[syncs] How to distinguish multiple source accounts?

### DISCUSSION THREAD

**Discussion by marionettest** [airbytehq/airbyte#33856] [Connector Questions]
We have multiple Klaviyo accounts as sources and want to synchronize them to a mysql database as destination. But after the synchronization is completed, we find that there seems to be no account information in the database. Is there any way to distinguish different source accounts in each row of data in destination?

**Comment by marcosmarxm** [comment]
No, there isn't a way to inject a key-value from a source to the destination. You can use a prefix to identify different accounts or the custom namespace to store different account into different schemas.

Can you provide more info about your use case? Are you sending multiple sources to the same destination with the same namespace and prefix?

**Comment by evantahler** [comment]
Do not configure Airbyte so that multiple syncs write to the same tables - that will probably lead to data loss, especially when a reset or schema migration occurs.

One thing you could do is first import data from your multiple Klavio accounts into multiple tables, prefixed with the account name (you can set this up via a "destination namespace" in Airbyte).  You'll end up with tables like `account1_user`, `account2_user`, etc.  

Then, you can create a view or use a tool like dbt to transform all of those tables into one, using the table prefix as the content for the additional column `account` you want to fill in.

[a comment by an automated account is not shown]
